src/components/card/Card.js

```diff
@@ -1,87 +1,50 @@
 import styled from "styled-components";
-import useLocalStorage from "../../hooks/useLocalStorage";
-import ActivityCards from "./ActivityCards";
-import AddActivity from "./AddActivity";
-import DeleteActivity from "./DeleteActivity";
-
-export default function Card() {
-  const [activities, setActivities] = useLocalStorage("activities", {
-    activityList: [
-      {
-        id: 1,
-      },
-    ],
-  });
-
-  const AddActivityHandler = (
-    activityID,
-    activityName,
-    activityDate,
-    activityCity,
-    activityTime
-  ) => {
-    setActivities({
-      activityList: [
-        ...activities.activityList,
-        {
-          id: activityID,
-          name: activityName,
-          date: activityDate,
-          city: activityCity,
-          time: activityTime,
-        },
-      ],
-    });
-  };
-
-  const DeleteActivityHandler = (activityName) => {
-    let index,
-      count = 0;
-    let activitiesList = [...activities.activityList];
-    activities.activityList.map((element) =>
-      element.name === activityName ? (index = count) : count++
-    );
-    activitiesList.splice(index, 1);
-    setActivities({ activityList: activitiesList });
-  };
 
+export default function Card({ id, activity, tags, onDelete, onToggle }) {
   return (
-    <>
-      <InputComponent>
-        <AddActivity
-          add={AddActivityHandler}
-          id={activities.activityList[activities.activityList.length - 1].id}
-        />
-        <DeleteActivity delete={DeleteActivityHandler} />
-      </InputComponent>
-
-      <AllCards>
-        {activities.activityList.map((element) => (
-          <ActivityCards
-            key={element.id}
-            name={element.name}
-            date={element.date}
-            city={element.city}
-            time={element.time}
-          />
+    <CardArticle>
+      <h2>{activity}</h2>
+      <TagContainer>
+        {tags.map((tag) => (
+          <Container key={Math.random()}>{tag}</Container>
```

Comment:
`key` props need to be unique and stable **across render cycles**. Your key is unique but not stable. Use ids or similar pieces of information for keys.